include/ktx.h

```diff
@@ -1172,12 +1172,12 @@ extern KTX_API const ktx_uint32_t KTX_ETC1S_DEFAULT_COMPRESSION_LEVEL;
 typedef struct ktxAstcParams {
     ktx_uint32_t structSize;
         /*!< Size of this struct. Used so library can tell which version
-             of struct is being passed.
-         */
+             of struct is being passed.*/
```

Comment:
Please be consistent over whether the closing `*/` is on the same line as the last sentence or on the next line. Some changes here move it to the same line, others move it from the same line to a new line.

Follow the style used in the ktxBasisParams structure.

Reply:
I clearly messed up because these are not part of my PR and I only changed those to be consistent.

Reply:
I think I know what happened. In the `ktxAstcParams` struct the closing `*/` is on same line while in the `ktxBasisParams` section its on new line. That way they are consistent so I left it at that. I would rather not open an other can of worms if I have to go one way or another for the whole header. At least locally they are consistent.

Reply:
Also happy to do 
> Follow the style used in the ktxBasisParams structure.